app/read_file_json.py

```diff
@@ -0,0 +1,14 @@
+from json import load
+
+from app.create_instance import create_customers, create_shops
+
+
+def read_json() -> tuple:
+    with open("app/config.json", "r") as file:
+        data = load(file)
+
+    fuel_price = data["FUEL_PRICE"]
+    customers = create_customers(data["customers"])
+    shops = create_shops(data["shops"])
+
+    return fuel_price, customers, shops
```

Comment:
no need to create separate file for this func
also u can implement create_shops\create_customers func inside custormer and shop classes

Reply:
I've talked to Yarik and he said I can add F821 to .flake8 because flake8 don't pass annotation as a literal